libnss-aad/docker/ubuntu/Dockerfile

```
@@ -0,0 +1,70 @@
+FROM ubuntu:18.04
```

Let's also rename and bump, e.g., `libnss_aad/ubuntu/Dockerfile`

```suggestion
FROM ubuntu:20.04
```